web/src/stream_popover.js

```diff
@@ -173,6 +173,7 @@ function build_stream_popover(opts) {
                 $(this).closest(".popover").fadeOut(500).delay(500).remove();
 
                 const sub = stream_popover_sub(e);
+                hide_stream_popover();
```

Comment:
Is this an existing bug we should fix via a preparatory PR?

Reply:
No no I accidentally added an unrelated commit while rebasing.